exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,73 @@
+#include <array>
+#include <string>
+
+#include "twelve_days.h"
+
+namespace twelve_days {
+
+std::string recite(int start_day, int end_day) {
+    constexpr std::array<const char*, 13> ordinals = {
```

Comment:
The compiler can derive the template arguments automatically:
```suggestion
    constexpr std::array ordinals = {
```